src/modules/Cart/Service.php

```diff
@@ -47,31 +47,84 @@ public function getSearchQuery($data)
      * @return \Model_Cart
      */
     public function getSessionCart()
-    {
-        $sqlBindings = [':session_id' => $this->di['session']->getId()];
-        $cart = $this->di['db']->findOne('Cart', 'session_id = :session_id', $sqlBindings);
-
-        if ($cart instanceof \Model_Cart) {
-            return $cart;
-        }
-
-        $cc = $this->di['mod_service']('currency');
-
-        if ($this->di['session']->get('client_id')) {
-            $client_id = $this->di['session']->get('client_id');
-            $currency = $cc->getCurrencyByClientId($client_id);
+    {   
+        $secret_key = $this->di['db']->getCell('SELECT value FROM setting WHERE param = :param', [':param' => 'ssh_key']);
```

Comment:
Neither `ssh_key` nor `ssh_iv` will ever be defined. They aren't default values and you've added nothing to set them